include/dba.php

```diff
@@ -504,6 +504,14 @@ function dbesc($str) {
 	}
 }
 
+function any_value_fallback($sql) {
+	//Considerations for Standard SQL, or MySQL with ONLY_FULL_GROUP_BY (default since 5.7.5).
+	//ANY_VALUE() is available from MySQL 5.7 https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html
+	//A standard fallback is to use MIN(), or nothing () in old MySQL 5.6-
+	//TODO: Skip this line when we know we are on a platform supporting ANY_VALUE()
+	return str_ireplace('ANY_VALUE(', 'MIN(', $sql);
```

Comment:
Additionally I'm not happy with this function. Doesn't it simply replaces the one text with the other one? So when we are discussing about this topic in Friendica then even inside of the body message the "any_value" will be replaced with "min".